cdap-proto/src/main/java/co/cask/cdap/proto/ProgramRunStatus.java

```diff
@@ -26,5 +28,26 @@
   SUSPENDED,
   COMPLETED,
   FAILED,
-  KILLED
+  KILLED;
+
+  /**
+   * Conversion from program run status to Workflow node status.
+   * @param status the program run status to be converted
+   * @return the converted Workflow node status
+   */
+  public static WorkflowNodeState.NodeStatus toNodeStatus(ProgramRunStatus status) {
+    switch(status) {
```

Comment:
Seems like `NodeStatus` is just a subset of `ProgramRunStatus`, is it? If that's the case, can we just use `ProgramRunStatus`?